unittests/test_a_c.py

```diff
@@ -8,10 +8,10 @@
     CoreKernels,
     CassiniKernels,
     ExtraKernels,
-    download_kernels,
+    checking_pathlike_filename_variants, download_kernels,
```

Comment:
Please put these on separate lines in all files

Reply:
Done.  My bad for letting PyCharm handle the imports without paying attention.